apps/cloud_composer/include/pcl/apps/cloud_composer/items/fpfh_item.h

```diff
@@ -74,7 +82,11 @@ namespace pcl
         pcl::PointCloud<pcl::FPFHSignature33>::Ptr fpfh_ptr_;
         double radius_;
         pcl::visualization::PCLPlotter::Ptr plot_;
+#if VTK_MAJOR_VERSION > 8
+        QVTKOpenGLNativeWidget* qvtk_;
```

Comment:
I feel like we should be using `std::unique_ptr` or based on the ctor, `std::shared_ptr` for this pointer as well as `model_` (out of purview of this PR)

Reply:
Want me to update in this PR or save it for another. I guess it will be required multiple places.

Reply:
What `model_` are you refering to?

Reply:
`ProjectModel* model_` in cloud_view.h

And in another PR please. Just that the usage of raw pointer stands out during refactoring. (it's not a smart pointer manager by VTK is it?)

Reply:
ProjectModel is a Qt raw pointer object. I'll mark this as resolved? and create a issue?